packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1632,3 +2128,294 @@ Specific property types support a subset of [JSON Schema validations](https://to
+      "href": "https://${yourOktaDomain}/api/v1/meta/schemas/group/default",
+      "method": "GET",
+      "rel": "self"
+    }
+  }
+}
+```
+
+### Schema properties
+
+The Group Schema is a valid [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04) document with the following properties:
+
+| Property    | Description                                                                              | DataType                                                           | Nullable | Unique | Readonly | Validation  |
+|:------------|:-----------------------------------------------------------------------------------------|:-------------------------------------------------------------------|:---------|:-------|:---------|:------------|
+| id          | URI of Group Schema                                                                      | String                                                             | FALSE    | TRUE   | TRUE     |             |
+| $schema     | JSON Schema version identifier                                                           | String                                                             | FALSE    | FALSE  | TRUE     |             |
+| name        | Name for the Schema                                                                      | String                                                             | FALSE    | TRUE   | TRUE     |             |
+| description | Description for the Schema                                                               | String                                                             | FALSE    | FALSE  | FALSE    |             |
+| title       | User-defined display name for the Schema                                                 | String                                                             | FALSE    | FALSE  | FALSE    |             |
+| created     | Timestamp when the Schema was created                                                    | [ISO 8601 String](https://tools.ietf.org/html/rfc3339)             | FALSE    | FALSE  | TRUE     |             |
+| lastUpdated | Timestamp when the Schema was last updated                                               | [ISO 8601 String](https://tools.ietf.org/html/rfc3339)             | FALSE    | FALSE  | TRUE     |             |
+| definitions | App User Profile subschemas                                                              | [App User Profile Subschemas](#app-user-profile-subschemas)        | FALSE    | FALSE  | FALSE    | JSON Schema |
+| type        | Type of [root Schema](https://tools.ietf.org/html/draft-zyp-json-schema-04#section-3.4)  | String                                                             | FALSE    | FALSE  | TRUE     |             |
+| properties  | User object properties                                                                   | [App User object](/docs/reference/api/apps/#application-user-object) property set | FALSE    | FALSE  | TRUE     |             |
```

Comment:
That might also be a copy paste error where I copied it from :trollface:  since it talks about a User object and an App User object in the same line.  I'll fix both of these.